src/Components/TodoClass.js

```diff
@@ -1,12 +1,22 @@
+import {action, makeObservable, observable} from 'mobx'
 class ToDoClass {
```

Comment:
same a avoid `class` in this name, and class are always going to be singular.

Reply:
Done and Understand